src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,9 @@
+server.port=8082
```

Comment:
I think this port is already taken by another Fog-service. Please change it to a higher number.